packages/react-components/react-breadcrumb/src/components/Breadcrumb/useBreadcrumb.ts

```diff
@@ -17,18 +18,28 @@ export const useBreadcrumb_unstable = (props: BreadcrumbProps, ref: React.Ref<HT
     dividerType = 'chevron',
     iconPosition = 'before',
     size = 'medium',
+    list,
     ...rest
   } = props;
+
+  const focusAttributes = useArrowNavigationGroup({
+    circular: true,
+    axis: 'horizontal',
+    memorizeCurrent: true,
+  });
+
   return {
     components: {
       root: 'nav',
       list: 'ol',
     },
     root: getNativeElementProps('nav', {
       ref,
+      'aria-label': 'breadcrumb' || props['aria-label'],
```

Comment:
I'm kinda iffy on whether we should provide a default label, though if we do then these need to be reversed so that `props['aria-label']` is used if it is defined. I also wonder if "breadcrumb" is recognizable to people who don't work in tech. Maybe something like this, though I'm not sold on "path":
```suggestion
      'aria-label': props['aria-label'] ?? 'path',
```

Reply:
In our design spec there's "Breadcrumb" aria-label. I'll talk to the designers

Reply:
For now I did this according to the design spec. But if more changes are needed I'll fix it in the following PRs